spotbugs/src/xsl/default.xsl

```diff
@@ -282,7 +282,7 @@
 		<xsl:copy-of select="$bugTableHeader"/>
 		<xsl:apply-templates select="$warningSet">
 			<xsl:sort select="@abbrev"/>
-			<xsl:sort select="Class/@classname"/>
+			<xsl:sort select="Class[1]/@classname"/>
```

Comment:
Does this still work if there is only one element?

Reply:
I think yes. At least I see in our build that both single / multiple matches are working now.
I have not much experience with that, but stack overflow suggests this as a viable approach:
https://stackoverflow.com/questions/16928121/how-to-select-first-node-only-in-xslt

Reply:
@mkienenb , @paulk-asert : WDYT?

Reply:
+1